src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -148,9 +149,12 @@ def direct_file_url(self,
     def format_version(cls, version: datetime.datetime) -> str:
         return version.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
 
-    def _run_sql(self, query):
+    def _run_query(self, query: str) -> BigQueryRows:
         return self.tdr.run_sql(query)
 
+    def _run_union_query(self, subqueries: Iterable[str]) -> BigQueryRows:
+        return self._run_query('UNION ALL'.join(subqueries))
```

Comment:
```suggestion
        return self._run_query(' UNION ALL '.join(subqueries))
```
Currently all queries start and end with whitespace however I don't think `_run_union_query` should assume it to be so.